### PR TITLE
Additional Stuff

### PR DESCRIPTION
**Pull request by shawntabrizi**
Some nits.

### CODE REVIEW

File: src/images/bow.svg [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (erler) [inline src/images/bow.svg:1]: We will get some custom visuals from either Iggy or Zsofia. No need to add this

File: src/pages/gift/about/FAQ.js [hunk 2]
```diff
@@ -9,9 +9,9 @@ export default function FAQ() {
   const q6 = 'I am having trouble claiming my gift.';
   const a1 = (
     <p className="pb-3">
-      Polkadot is a blockchain, that connects other blockchains. DOT is the
```
Comment (erler) [inline src/pages/gift/about/FAQ.js:12]: The exact language will probably get finalised by some of our writers. However, if our audience are users totally new to the ecosystem, the language used should probably be rather simple than overly technical

File: src/pages/gift/about/FAQ.js [hunk 3]
```diff
@@ -22,10 +22,10 @@ export default function FAQ() {
     </p>
   );
   const a2 =
-    'Polkadot Gifts lets you generate a unique hash that functions as a voucher to claim an amount of DOT. Send the unique gift secret to your friend or family member. They are guided through a simple account creation process and reveal the gift secret to the Polkadot network to claim their DOTs into their account.';
+    'Polkadot Gifts is powered by on chain logic provided by the Polkadot Network. This website is completely stateless and does not track any sensitive information. All interactions are done between you and the Polkadot blockchain, keeping your account, your funds, and the gift claiming process secure.';
```
Comment (erler) [inline src/pages/gift/about/FAQ.js:25]: I like this explanatory bit about not tracking. Still for someone new, we need a basic explanation of why you want to use this app in the first place

File: src/pages/gift/about/FAQ.js (hunk 2 above)
Comment (hamidra) [reply src/pages/gift/about/FAQ.js:12]: To me, there will be two groups of users that are going to use this app. The people that are sending the gifts who are more likely blockchain savvy and have used polkadot before and on the other side the gift recipients who are new to the network and probably have not used polkadot or any other blockchains at all. The content creation team can hopefully help to keep the balance in targeting the both group of audiences. 
I also have this item #8  open to add some more info regarding account creation that should be targeted for the users that are new to the ecosystem.

**Review by hamidra — APPROVED** [review]

**Comment by shawntabrizi** [comment]
@hamidra @erler feel free to discard or keep any changes in this PR and then merge what you like

File: src/pages/gift/about/FAQ.js (hunk 2 above)
Comment (erler) [reply src/pages/gift/about/FAQ.js:12]: Yes, I agree with the two different groups of users, and need to balance language!

**Comment by hamidra** [comment]
Thanks all! Merging. We can update the faqs with more info later when we have feedback from design and content team.

**Pull request merged**